apps/frontend/src/contexts/AuthContext.tsx

```diff
@@ -103,14 +102,6 @@ export function AuthProvider({
 
   const value = { session, loading, googleLogin, githubLogin, logout }
 
-  if (loading) {
-    return (
-      <div className="flex h-screen items-center justify-center bg-white dark:bg-[#313131]">
-        <Loader />
-      </div>
-    )
-  }
-
```

Comment:
You can add your loader here if you want

Reply:
Yeah, for some reason the loading.tsx was causing the build to fail. I still have no clue why it was doing that but now I switched back to the previous loading logic